src/components/navbars/DesktopNav.jsx

```diff
@@ -1,32 +1,39 @@
+import ProfileNav from './ProfileNav';
 import loginIcon from '../../assets/icons/login.png';
 import searchIcon from '../../assets/icons/search.png';
+import { useAuth } from '../../AuthProvider';
 
 const DesktopNav = ({ onSearch, onLogin, onJoin, stateSearch, stateLogin, stateJoin }) => {
+    const { isLoggedIn } = useAuth()
     return (
-        <nav className="w-2/3 hidden relative py-6 sm:flex justify-between">
+        <nav className="w-2/3 hidden relative py-6 sm:flex justify-between items-center">
             {
                 stateSearch
                     ? <form className="border-2 border-red rounded-full h-10 w-2/4 flex justify-between items-center px-4 ml-6 gap-1">
-                        <input className='focus:outline-none w-2/3' aria-label="Search" placeholder="Search online classes or teachers..." />
+                        <input className='focus:outline-none w-2/3' aria-label="Search" placeholder="Search classes or teachers..." />
                         <button onClick={onSearch} aria-label="Search"><img src={searchIcon} className="h-6" alt="search icon" /></button>
```

Comment:
The search bar can be it's own component. It'll help clean up this component a lot. I believe it's also used in another component.

Reply:
I think the team decided that the search component would have its own search bar and the value would be passed from one to another.